owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/FileListAdapter.kt

```diff
@@ -304,6 +304,8 @@ class FileListAdapter(
             if (file.isFolder) {
                 // Folder
                 fileIcon.setImageResource(R.drawable.ic_menu_archive)
+            } else if (file.mimeType == "text/uri-list") {
```

Comment:
Better to use a constant as you did in `FileDisplayActivity` to keep consistency

Reply:
Done